README.md

```diff
@@ -2,10 +2,10 @@ Apache Pekko
 ============
 
 Apache Pekko is an open-source framework for building applications that are concurrent, distributed, resilient and elastic.
+Pekko uses the Actor Model to provide more intuitive high-level abstractions for concurrency.
+Using these abstractions, Pekko also provides libraries for persistence, streams, HTTP, and more.
```

Comment:
could you remove the stray comma after `HTTP`?

Reply:
We're just nitpicking here, but to me a serial comma seems more natural before "and more". I see it as basically the same as "etc.", where the prevalent style is to use a comma before.

Reply:
fair enough, I never use a comma before `and` but I understand why it can be used here